core/src/backend/ui.rs

```diff
@@ -15,6 +15,9 @@ pub trait UiBackend {
     /// Changes the mouse cursor image.
     fn set_mouse_cursor(&mut self, cursor: MouseCursor);
 
+    /// Get the clipboard content
+    fn clipboard_content(&mut self) -> String;
```

Comment:
`clipboard_content()` can have an immutable `&self` recevier, right?

Reply:
Arboard's Clipboard.get_text function takes a mutable &self, so this doesn't seem to work